src/views/components/item.jsx

```diff
@@ -51,7 +50,7 @@ export default class Item extends React.Component {
                 <div className="actionArea">
                     <img src="../assets/images/add.svg"/>
                     <img src="../assets/images/edit.svg"/>
-                    <img src="../assets/images/delete.svg"/>
+                    <img onClick={deleteItem.bind(this, this.state.item.address)} src="../assets/images/delete.svg"/>
```

Comment:
I don't see a reason to prevent checked out items from being deleted.  If someone does try to check in a deleted item, then we can just have a message saying: "That item was deleted or does not exist" and move on.  Additionally, if someone were to steal an item, and Tech Support had no chance of getting it back, we would want to allow them to delete it from the inventory.